README.md

```diff
@@ -1 +1,31 @@
-# ODP
\ No newline at end of file
+# E2E-Test Framework
+
+##1. How to run the e2e test against :
+
+### a) Local CDAP sandbox:
+
+Provide the local CDAP sandbox URL in [connectionParameters.properties](src/main/resources/connectionParameters.properties) file.
+>cdfurl=http://localhost:11011/pipelines/ns/default/studio
+
+Other keys (like @TC-Demo-1_GCS, Data-Set, etc) can be modified as needed. <br>
+
+**Configuration Details required for Setup:**
```

Comment:
This section describes how to add e2e tests in a plugin repo. I think it should be moved to Addnewplugin.md.

Reply:
Moved.